contracts/BodhiToken.sol

```diff
@@ -7,11 +7,16 @@ contract BodhiToken is StandardToken, Ownable {
   // Token configurations
   string public constant name = "Bodhi Token";
   string public constant symbol = "BOT";
-  uint256 public constant decimals = 18;
+  uint256 public constant decimals = 8;
```

Comment:
Since now getTokenExchangeRate will accept dynamic `decimals` and `nativeDecimals` as parameters, we can change back this to 18 for saving confusion when this contract gets audited by 3rd party.

Reply:
done